clang/lib/CodeGen/CGExprAgg.cpp

```diff
@@ -135,6 +135,17 @@ class AggExprEmitter : public StmtVisitor<AggExprEmitter> {
     EnsureDest(E->getType());
 
     if (llvm::Value *Result = ConstantEmitter(CGF).tryEmitConstantExpr(E)) {
+      // An empty record can overlap other data (if declared with
+      // no_unique_address); omit the store for such types - as there is no
```

Comment:
Candide question: Empty record still need one byte when their address is taken (thus this comment about `no_unique_address` I guess), why don't we see that in the diff?

Reply:
See what, exactly?  Given the derived class, computing the address of the base class doesn't take any instructions, because it's the same address.